City-of-Pori/pori-kada
# PORI-X Removed unused modules

To test:

1. drush cc all
2. Go to features page and make sure there is no longer listed pori_domains, pori_media and pori pages features. They were already uninstalled from the system.
